Dynamic/Collections/Page.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Sulu\Bundle\FormBundle\Dynamic\Collections;
+
+use Sulu\Bundle\FormBundle\Dynamic\FormCollectionTitleInterface;
+use Sulu\Bundle\FormBundle\Dynamic\FormFieldTypeConfiguration;
+use Sulu\Bundle\FormBundle\Entity\FormField;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * TODO: add interace
+ * TODO: comments
+ */
+class Page implements FormCollectionTitleInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function CollectionTitleProvider() {
```

Comment:
I have renamed this method to the original name and updated the todo description.